### PR TITLE
Update gridstack.ts

### PR DESCRIPTION
**Pull request by ZOUJIELynn**
fix when set minRow greater than currentRow ,  the top value of placeholder is not set

### Description
Please explain the changes you made here. Include an example of what your changes fix or how to use the changes.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing (`yarn test`)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/gridstack.ts [hunk 1]
```diff
@@ -327,7 +327,7 @@ export class GridStack {
       maxRow: this.opts.maxRow,
       onChange: (cbNodes) => {
         let maxH = 0;
```
Comment (adumesny) [inline src/gridstack.ts:329]: just set 
let maxH = this.opts.minRow || 0; 
instead so it doesn't check every time AND makes sure it not undefined

Comment (adumesny) [reply src/gridstack.ts:329]: code might be incorrect if undefined ? (or max(0, undefined) happens to work...

**Comment by ZOUJIELynn** [comment]
> 
I set minRow =3 , but actully only two row has widgets , when i drag external widget at row 3, the top value of placeholder is not set, because there is no Css for gs-y=2

**Comment by adumesny** [comment]
seems like the right fix is to make sure the dragged in item DOES increase the maxH computation instead to update the CSS rules. Separate from having minRow set as it's possible to drag an item to the bottom anyway (past the current last item).

Can you post running example by any chance ? thanks!

**Comment by ZOUJIELynn** [comment]
> 

thanks , let me have a try

**Pull request closed without merging**